app/serializers/web_hook_user_serializer.rb

```diff
@@ -0,0 +1,33 @@
+class WebHookUserSerializer < UserSerializer
+  def can_edit
+    false
+  end
+
+  def include_email?
```

Comment:
I wonder if we should just be explicit here and include what we want instead of excluding since the payload size is the concern. Overtime, people will add more stuff to `UserSerializer` and it'll cause the web hook payload to bloat without anyone knowing about it